### ISSUE TITLE
<input> cannot be child of <p>

### ISSUE THREAD

**Issue by whitetrefoil** [MananTank/validate-html-nesting#8]
```
> require('validate-html-nesting').isValidHTMLNesting('p', 'input')
false
```
Is there any spec update forbid this usage? Checked the spec (https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element) and didn't see anything forbid this.

**Comment by MananTank** [comment]
@whitetrefoil Yeah, you're right. `p > input` is valid

**Comment by MananTank** [comment]
Fixed in 1.2.2
